### PR TITLE
chore: add Adarsh

### PR DESCRIPTION
**Pull request by adarrssh**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/916"></a>

### CODE REVIEW

File: public/data/Adarsh.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        "url": "https://github.com/adarrssh",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/Adarsh87498341",
+        "icon": "twitter"
+      }
+    ]
+  }
```
Comment (Panquesito7) [inline public/data/Adarsh.json:18]: Missing an endline.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/Adarsh.json (hunk 1 above)
Comment (krishguptadev) [inline public/data/Adarsh.json:18]:
```suggestion
}
```

File: public/data/Adarsh.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Adarsh",
+    "type": "personal",
+    "bio": "I am a web developer and an open source enthusiast.",
```
Comment (krishguptadev) [inline public/data/Adarsh.json:4]:
```suggestion
    "bio": "I am a web developer and an open-source enthusiast.",
```

**Review by syedsohan — APPROVED** [review]

File: public/data/Adarsh.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        "url": "https://github.com/adarrssh",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/Adarsh87498341",
+        "icon": "twitter"
+      }
+    ]
+  }
```
Comment (Pradumnasaraf) [inline public/data/Adarsh.json:18]:
```suggestion
{
  "name": "Adarsh",
  "type": "personal",
  "bio": "I am a web developer and an open-source enthusiast.",
  "avatar": "https://github.com/adarrssh.png",
  "links": [
    {
      "name": "Follow me on GitHub",
      "url": "https://github.com/adarrssh",
      "icon": "github"
    },
    {
      "name": "Follow me on Twitter",
      "url": "https://twitter.com/Adarsh87498341",
      "icon": "twitter"
    }
  ]
}
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Indentation Issue.

**Review by emmalearnscode — APPROVED** [review]

**Review by krishguptadev — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
LGTM, Thanks for contributing to **LinkFree**

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**